helix-view/src/document.rs

```diff
@@ -333,6 +334,7 @@ impl Document {
             encoding,
             text,
             selections: HashMap::default(),
+            selections_history: HashMap::default(),
```

Comment:
Rather than `_history` which makes it sound like `view.jumps` (previous selection stack for ctrl-i/ctrl-o), maybe something like `object_selections`?

Reply:
Also, `View` might be a better place for this, since you just need a `Vec<_>` instead of `HashMap<ViewId, _>`